test/fixtures/gcs-fixture/build.gradle

```diff
@@ -26,8 +26,8 @@ dependencies {
   api project(':server')
 }
 
-preProcessFixture {
-  dependsOn jar, configurations.runtimeClasspath
+tasks.named("preProcessFixture").configure {
+  dependsOn tasks.named("jar"), configurations.runtimeClasspath
```

Comment:
Looks like `jar` and `runtimeClasspath` should also be inputs here. In fact we should just do that and then rely on the implicit dependencies.

Reply:
fixed